float64 water
float64 gas_bubbles
float64 contaminants
float64 iodine_level
float64 pressure 
float64 temperature

